### PR TITLE
Add store split for Get Dashboard version method

### PR DESCRIPTION
**Pull request by idafurjes**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by mildwonkey — APPROVED** [review]
LGTM! 

Question for my own PRs, not this one: I see that you moved some types out of the `models` package; should I also be doing that in my dashboard PRs? Do we have a plan for what should stay in `models` vs. move?

**Comment by idafurjes** [comment]
@mildwonkey In general for split store we do, only the dashboard split store that you're working on was started by enterprise, it does not follow our POC. I think for now it is ok to leave them in model.

File: pkg/api/dashboard_test.go [hunk 1]
```diff
@@ -767,16 +796,20 @@ func TestDashboardAPIEndpoint(t *testing.T) {
...
 			}}
-
+		mockSQLStore := mockstore.NewSQLStoreMock()
 		restoreDashboardVersionScenario(t, "When calling POST on", "/api/dashboards/id/1/restore",
-			"/api/dashboards/id/:dashboardId/restore", dashboardService, cmd, func(sc *scenarioContext) {
+			"/api/dashboards/id/:dashboardId/restore", dashboardService, fakeDashboardVersionService, cmd, func(sc *scenarioContext) {
+				sc.dashboardVersionService = fakeDashboardVersionService
+
+				// restoreDashboardVersionScenario(t, "When calling POST on", "/api/dashboards/id/1/restore",
+				// 	"/api/dashboards/id/:dashboardId/restore", dashboardService, cmd, func(sc *scenarioContext) {
```
Comment (zserge) [inline pkg/api/dashboard_test.go:812]: Some leftovers, I suppose?

File: pkg/services/dashboardversion/dashverimpl/store.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	db db.DB
+}
+
+func (s *sqlStore) Get(ctx context.Context, query *dashver.GetDashboardVersionQuery) (*dashver.DashboardVersion, error) {
+	var version dashver.DashboardVersion
+	err := s.db.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		has, err := sess.Where("dashboard_version.dashboard_id=? AND dashboard_version.version=? AND dashboard.org_id=?", query.DashboardID, query.Version, query.OrgID).
+			Join("LEFT", "dashboard", `dashboard.id = dashboard_version.dashboard_id`).
+			Get(&version)
+
```
Comment (zserge) [inline pkg/services/dashboardversion/dashverimpl/store.go:26]: Good for now, but just to think about - could we make an in-memory JOIN instead (using the dashboard service, once that is complete)? Ideally we shouldn't access any other tables but the ones owned by the dashver service itself.

**Review by zserge — APPROVED** [review]

**Pull request merged**